feat/gene/gene_test.go

```diff
@@ -233,6 +242,26 @@ var transcriptTests = []struct {
 		ExonicLen: 1200,
 	},
 	{
```

Comment:
Don't see any reason for this. Sounds more like a test for `BaseOrientationOf()`. Please let me know if you have sth specific in mind.